### ISSUE TITLE
Compile Java code with -target

### ISSUE THREAD

**Issue by puffnfresh** [typelead/eta#328]
I had a go at running a Jar through "dx" for Android. It failed because the class file format was Java 8 (major version 52)

We should either:

1. Allow configuring the target
2. Have the Java target be the same as what Eta targets

**Comment by rahulmutt** [comment]
Agreed. The `eta` compiler should take a `-target` flag with the same format as javac. This flag should be used in the codegen when generating class files and should also be sent to `javac` when compiling `.java` files.

**Comment by rahulmutt** [comment]
The current workaround you can do is used the flag `-optjavac="-target [target]"`.
